src/FilterService/Controllers/UserController.cs

```diff
@@ -15,9 +15,10 @@ public class UserController : ControllerBase
     public async Task<FindResultResponse<UserInfo>> GetAsync(
       [FromServices] IFilterUsersCommand command,
       [FromQuery] UserFilter filter,
-      [FromQuery] PaginationValues value)
+      [FromQuery] PaginationValues value,
+      [FromBody] UsersSearchParameters parameters)
```

Comment:
body in http get request - maybe make it post or get UsersSearchParameters from query?